src/BaGet.AWS/S3StorageService.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.S3;
+using Amazon.S3.Model;
+using BaGet.AWS.Configuration;
+using BaGet.Core.Entities;
+using BaGet.Core.Services;
+using Microsoft.Extensions.Options;
+using NuGet.Versioning;
+
+namespace BaGet.AWS
+{
+    public class S3StorageService : IStorageService
+    {
+        private readonly string _bucket;
+        private readonly string _prefix;
+        private readonly AmazonS3Client _client;
+        private const string Seperator = "/";
```

Comment:
`Separator`

Reply:
Would it be possible to make an editorconfig ?